docs/extensions/bundled/activation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Activation
+level: advanced
+---
+Activation
+==========
+
+Once your Bundle can be autoloaded then one more step is needed to
```

Comment:
There should be a comma before 'then', right? 

I can't english, so i'm not sure. :-)